File: app/src/main/java/com/juanpablofajardo/postsapp/callbacks/PostsListener.java
package com.juanpablofajardo.postsapp.callbacks;

import com.juanpablofajardo.postsapp.objects.Post;

import java.util.List;

/**
 * Created by Juan Pablo Fajardo Cano on 4/29/18.
 */
public interface PostsListener extends ServiceEvent {
    void onPostsFetchSuccess(List<Post> posts);
}

File: app/src/main/java/com/juanpablofajardo/postsapp/callbacks/PostsListener.kt
package com.juanpablofajardo.postsapp.callbacks

import com.juanpablofajardo.postsapp.objects.Post

/**
 * Created by Juan Pablo Fajardo Cano on 4/28/18.
 */
interface PostsListener: ServiceEvent{
    fun onPostsFetchSuccess(posts: List<Post>)
}